server/spinwick.go

```diff
@@ -440,8 +445,8 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 	if withLicense {
 		installationRequest.License = s.Config.SpinWickHALicense
 	}
-	if envVars != nil && len(envVars) > 0 {
-		installationRequest.MattermostEnv = envVars
+	if len(envVars) > 0 {
+		installationRequest.PriorityEnv = envVars
```

Comment:
Nice improvement going with priority env 👍

There may be some edge cases here in regards to update and reset env vars. In particular you would need to be able to provide an empty env var object to clear everything out. This probably isn't super important right now, but just wanted to note it.

Reply:
Thanks @gabrieljackson. I think we are (at least partially) covering this through the `-clear-env` handling here:

https://github.com/mattermost/matterwick/blob/95ff4945aea3293930f0b4d8d20c17b013ede4a1/server/slash_commands.go#L148-L153

I actually used the cloud plugin source as a working example since we are effectively mimicking part of its functionality.

Hopefully, we'll get some feedback soon as devs start using this new command. I'd happily review any of this to accommodate our use cases.